packages/venia-concept/src/__mocks__/@magento/peregrine.js

```diff
@@ -0,0 +1,8 @@
+import { createElement } from 'react';
+
+/**
+ * the Price component from @magento/peregrine
+ * has browser-specific functionality and cannot
+ * currently by rendered in the test environment
+ */
+export const Price = () => <div/>;
```

Comment:
@jimbo I managed to acheive mock the Price component in this manner, which seems really clean as it gives complete control in a deterministic place (the `__mocks__` folder)